Tasks/HelmDeployV0/src/helm.ts

```diff
@@ -78,10 +76,11 @@ function runHelmSaveCommand(helmCli: helmcli, kubectlCli: kubernetescli, failOnS
 
 async function run() {
     var command = tl.getInput("command", true).toLowerCase();
+    var connectionType = tl.getInput("connectionType", true);
     var isKubConfigRequired = isKubConfigSetupRequired(command);
     var kubectlCli: kubernetescli;
     if (isKubConfigRequired) {
-        var kubeconfigfilePath = command === "logout" ? tl.getVariable("KUBECONFIG") : await getKubeConfigFile();
+        var kubeconfigfilePath = (command === "logout" || connectionType === "None") ? tl.getVariable("KUBECONFIG") : await getKubeConfigFile();
```

Comment:
When connectionType is none and customers are using external auth like kubelogin, a kubeconfig file path would be required to initiate the kubernetesCli here. 
A build test passed with setting KUBECONFIG=~/kube/config in my local build agent.
https://mwittest.visualstudio.com/merlynop/_build/results?buildId=570&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a